test/Riok.Mapperly.Tests/Mapping/EnumNamingStrategyTest.cs

```diff
@@ -0,0 +1,172 @@
+namespace Riok.Mapperly.Tests.Mapping;
+
+public class EnumNamingStrategyTest
+{
+    [Fact]
+    public void EnumToStringWithPascalCaseNamingStrategy()
```

Comment:
Add a test which uses the `MapEnumAttribute.FallbackValue`.

Reply:
Yes, I have added a test that uses MapEnumAttribute.FallbackValue.